### DISCUSSION TITLE
metadata contents

### DISCUSSION THREAD

**Discussion by cburgard** [hep-statistics-standard/hep-statistics-standard#29] [Standard]
Hi everyone!

I'm going through some past comments we have received a while back. 

I have a set of comments from @kratsg relating to the contents of `metadata`

- Metadata for publications should be an array of dictionaries, rather than an array of strings. 
- What should the format of the identifier be in cases where no DOI is available? We could, for example, target InspireHEP or HEPdata identifiers 
- we should maybe structure the `metadata` field around the `Citation.cff` format [link](https://citation-file-format.github.io/)
- should there be any restriction on the allowed set of package names?

**Comment by jonas-eschle** [comment]
Array of dicts sounds general, what metadata do we expect? Maybe we can collect use-cases here.

**Comment by sabinekraml** [comment]
Re the point "What should the format of the identifier be in cases where no DOI is available? We could, for example, target InspireHEP or HEPdata identifiers" above: HEPData actually issues DOIs. In any case, using InspireHEP identifiers should be a good idea, perhaps even as a dedicated component, as HEPData uses those identifiers for cross-referencing. 

The current specification of the "publications" component as an "array of document identifiers of publications associated with this file" is very vague; it would be good to make this more specific, to facilitate interoperability of tools that use these metadata.  

As for other matadata,in my opinion "authors" should be a required, not an optional component (there needs to be somebody responsible for any kind of publication or data product). And I'd like to suggest a "contact" component, where one would give, e.g., the generic email address of the relevant ATLAS/CMS physics group conveners, of the physics working group, or of whoever produced the statistical model.

**Reply by cburgard** [reply]
Hey @sabinekraml !
These are excellent points and I agree with them whole-heartedly.
Would you be willing to make a concrete proposal on how the metadata should look in your opinion to be maximally useful?

**Reply by kratsg** [reply]
Not everyone who uses ROOT (or wants to serialize) will have InspireHEP identifiers (or even HEPData) will they? The presumption will be that the initial users of course will be HEP-affiliated, but I imagine that HS3 should be for not just HEP alone...

**Comment by sabinekraml** [comment]
Hi,

I'm currently on vacation but will make a proposal once back at work.
(Remind me if I forget and you haven't heard from me by the end of August)

Cheers, Sabine

On Tue, Aug 6, 2024 at 1:32 PM Carsten Burgard ***@***.***>
wrote:

> Hey @sabinekraml <https://github.com/sabinekraml> !
> These are excellent points and I agree with them whole-heartedly.
> Would you be willing to make a concrete proposal on how the metadata
> should look in your opinion to be maximally useful?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hep-statistics-serialization-standard/hep-statistics-serialization-standard/discussions/29#discussioncomment-10252821>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG3ROODINWKJV5DPP5CGCXDZQCX5BAVCNFSM6AAAAABAFYTKOWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRVGI4DEMI>
> .
> You are receiving this because you were mentioned.Message ID:
> <hep-statistics-serialization-standard/hep-statistics-serialization-standard/repo-discussions/29/comments/10252821
> @github.com>
>


-- 
_______________________________________________________
Sabine Kraml  - ***@***.*** -  www.kraml.net

LPSC
53 Av des Martyrs
38026 Grenoble
France
(+33)(0)4 76 28 40 52
